wagtail_localize/templates/wagtail_localize/admin/edit_translation.html

```diff
@@ -35,10 +35,22 @@
 {% block extra_js %}
     {% include "wagtailadmin/pages/_editor_js.html" %}
 
-    <script type="text/javascript" src="{% url 'wagtail_localize:javascript_catalog' %}"></script>
-    <script type="text/javascript" src="{% versioned_static 'wagtailadmin/js/page-chooser-modal.js' %}"></script>
-    <script type="text/javascript" src="{% versioned_static 'wagtailimages/js/image-chooser-modal.js' %}"></script>
-    <script type="text/javascript" src="{% versioned_static 'wagtaildocs/js/document-chooser-modal.js' %}"></script>
-    <script type="text/javascript" src="{% versioned_static 'wagtailsnippets/js/snippet-chooser-modal.js' %}"></script>
-    <script type="text/javascript" src="{% versioned_static 'wagtail_localize/js/wagtail-localize.js' %}"></script>
+    <script src="{% url 'wagtail_localize:javascript_catalog' %}"></script>
+
+    {% if has_legacy_styling %}
+        <script src="{% versioned_static 'wagtailadmin/js/page-chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtailimages/js/image-chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtaildocs/js/document-chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtailsnippets/js/snippet-chooser-modal.js' %}"></script>
+    {% else %}
+        <script src="{% versioned_static 'wagtailadmin/js/page-chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtailadmin/js/page-chooser.js' %}"></script>
+        <script src="{% versioned_static 'wagtailimages/js/image-chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtailimages/js/image-chooser.js' %}"></script>
+        <script src="{% versioned_static 'wagtaildocs/js/document-chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtaildocs/js/document-chooser.js' %}"></script>
+        <script src="{% versioned_static 'wagtailadmin/js/chooser-modal.js' %}"></script>
+        <script src="{% versioned_static 'wagtailsnippets/js/snippet-chooser.js' %}"></script>
```

Comment:
considering we redirect users to edit choosers, rather than allow them to choose another one I am not sure we need these, but that's for another PR